utils/src/main/scala/com/salesforce/op/test/MemoryAppender.scala

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
define this appender in the same file as `OpSparkListenerTest` - we dont need to expose it as it's currently only useful in one test.